### PR TITLE
Add PowerVS cloud controller manager

### PR DESCRIPTION
**Pull request by Karthik-K-N**
This PR contains the changes to add Power VS cloud controller manager

### CODE REVIEW

File: manifests/0000_26_cloud-controller-manager-operator_01_images.configmap.yaml
Comment (JoelSpeed) [inline manifests/0000_26_cloud-controller-manager-operator_01_images.configmap.yaml:21]: The rest of the providers in this list are alphabetical, can we move this to the correct place alphabetically

File: manifests/image-references
Comment (JoelSpeed) [inline manifests/image-references:44]: Is this image in payload yet?

Comment (mkumatag) [reply manifests/image-references:44]: not yet, lets wait
/hold

Comment (mkumatag) [reply manifests/image-references:44]: The [ART ticket](https://issues.redhat.com/browse/ART-3885) is closed and image will get into release image post this image-reference addition I feel.

The sample image looks like:
```
registry-proxy.engineering.redhat.com/rh-osbs/openshift4-ose-powervs-cloud-controller-manager:v4.11.0-202203221643.p0.ga6bfa07.assembly.stream 
```

**Comment by mkumatag** [comment]
why bot is not listening to me!
/meow

File: manifests/0000_26_cloud-controller-manager-operator_01_images.configmap.yaml
Comment (Karthik-K-N) [reply manifests/0000_26_cloud-controller-manager-operator_01_images.configmap.yaml]: Updated it in the latest commit

File: pkg/cloud/powervs/assets/deployment.yaml
Comment (mkumatag) [inline pkg/cloud/powervs/assets/deployment.yaml:1]: everywhere we need to change the reference from `ibm-cloud-controller-manager` to `powervs-cloud-controller-manager`

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Review by mkumatag — APPROVED** [review]
/lgtm
/label tide/merge-method-squash

**Comment by JoelSpeed** [comment]
I've removed the merge-method-squash as there is currently a bug in our release log generator that breaks when using this tool. The label should only be used on non-payload repos

**Comment by mkumatag** [comment]
> I've removed the merge-method-squash as there is currently a bug in our release log generator that breaks when using this tool. The label should only be used on non-payload repos

ah! thanks for that, I didn't know about this! may be will merge the code for now if no further comments.

/unhold

**Comment by mkumatag** [comment]
I see the failures are related to quota and there are some changes made in the deprovisioning, may be we need to wait for some more time to see that in action, will revisit may be tomorrow or later unless @JoelSpeed is okay to skip those failing jobs ;)

**Pull request merged**
